scripts/ingestion/database/maintainer_db_init.py

```diff
@@ -1,8 +1,12 @@
 #!/usr/bin/env python
 import sqlite3
 
-conn = sqlite3.connect('/data/yellow/vineet/database/bi_multi_tables.db')
-cur = conn.cursor()
+def target_con(con):
+        conn = sqlite3.connect(con)
+        cur = conn.cursor()
+        return cur
+
+target_con('/data/yellow/vineet/database/bi_multi_tables.db')
```

Comment:
same as above, you amy want to use the same variable (perhaps under a separate file called `constants.py`)

Reply:
declared it as constant under constants.py

Reply:
Declared